### PR TITLE
added improvements

### PR DESCRIPTION
**Pull request by trba1810**
Improvements that was discussed here: https://github.com/anicmarko9/responsive-front/pull/123

### CODE REVIEW

File: src/components/Shop/Product.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <div className="max-w-sm rounded overflow-hidden shadow-lg bg-green-100">
+      <figure>
+        <img
+          className="mx-auto w-9/12 md:h-[300px] rounded-lg"
+          src={imgSrc || notFound}
+          alt={title}
+        />
+      </figure>
+      <div className="px-6 py-4">
+        <figcaption className="font-bold text-xl mb-2">{title}</figcaption>
```
Comment (anicmarko9) [inline src/components/Shop/Product.tsx:23]: You misused `<figcaption>` here, please read this documentation: [figcaption](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption). Either use `<h2>` or put this inside of `<figure>`, but then be careful with CSS styling.

File: src/components/Shop/Product.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const Product = ({
+  title,
+  description,
+  imgSrc,
+  price,
+}: IProduct): JSX.Element => {
+  // A fallback image source in case the provided image is not found.
+  const notFound = '../ImgNotFound.svg';
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg bg-green-100">
```
Comment (anicmarko9) [inline src/components/Shop/Product.tsx:14]: `<div>` should be changed to either `<article>` or `<section>`.

**Review by anicmarko9 — COMMENTED** [review]
Thanks for fixing, but there are two smaller misused HTML tags.

**Comment by trba1810** [comment]
I've made changes.

File: src/components/Shop/Product.tsx (hunk 2 above)
Comment (jakepears) [reply src/components/Shop/Product.tsx]: it's fine to use a div. doesn't make too much of a difference as long as it's styled

**Pull request merged**